hicommonwealth/commonwealth
# 5398-adminGrowl

## Link to Issue
Closes: #5398 and #5805

## Description of Changes
- Added Growl for groups / gating

## Test Plan
- Go to the discussion section 
- Make sure Create a group / View groups go to the intended pages
- Learn More should goto the gating blog post
- Ensure the "Please don't show this again" checkbox works as intended. 


[Figma](https://www.figma.com/file/ePuGFatYQI6GL72i27sVwj/Gating?type=design&node-id=1784-312&mode=dev)
